utilities/test_suite/common.py

```diff
@@ -310,15 +310,15 @@ def print_performance_tests_summary(logFile, functionalityGroupList, numRuns):
             prevLine = line
 
     # Print log lengths
-    print("Functionalities - "+ str(funcCount))
+    print("Functionalities - " + str(funcCount))
 
     # Print summary of log
-    print("\n\nFunctionality\t\t\t\t\t\tFrames Count\tmax(ms/batch)\t\tmin(ms/batch)\t\tavg(ms/batch)\n")
+    header_format = "{:<70} {:<15} {:<15} {:<15} {:<15}"
+    row_format = "{:<70} {:<15} {:<15} {:<15} {:<15}"
+    print("\n" + header_format.format("Functionality", "Frames Count", "max(ms/batch)", "min(ms/batch)", "avg(ms/batch)") + "\n")
     if len(functions) != 0:
-        maxCharLength = len(max(functions, key = len))
-        functions = [x + (' ' * (maxCharLength - len(x))) for x in functions]
         for i, func in enumerate(functions):
-            print(func + "\t" + str(frames[i]) + "\t\t" + str(maxVals[i]) + "\t" + str(minVals[i]) + "\t" + str(avgVals[i]))
+            print(row_format.format(func, str(frames[i]), str(maxVals[i]), str(minVals[i]), str(avgVals[i])))
```

Comment:
Please change variable name from snake case  to camel case
row_format to rowFormat
header_format to headerFormat

Reply:
Done